pkg/k8shandler/kibana/consolelink.go

```diff
@@ -0,0 +1,50 @@
+package kibana
+
+import (
+	consolev1 "github.com/openshift/api/console/v1"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func NewConsoleLink(name, namespace, href string) *consolev1.ConsoleLink {
+	return &consolev1.ConsoleLink{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: namespace,
+		},
+		Spec: consolev1.ConsoleLinkSpec{
+			Location: consolev1.ApplicationMenu,
+			Link: consolev1.Link{
+				Text: "Logging",
```

Comment:
Seems a bit vague... maybe `Kibana logging URL` ?

Reply:
Based on: https://github.com/openshift/console/blob/fa8a554c0ac4d7e20423165a8f4457311ace2d46/frontend/public/components/nav/admin-nav.tsx#L85

Reply:
Or "Kibana Logging". 
Are there logging alternatives commonly used?  I don't know off the top of my head, so not sure the risk of ambiguity here.

Reply:
So far we support only Kibana. TBH, the ConsoleLinks replace the old configmap, so the rendered output should stay the same, right?